lib/taskana-simplehistory-example/src/main/java/pro/taskana/simplehistory/ExampleBootstrap.java

```diff
@@ -1,23 +1,43 @@
-package pro.taskana.history.plugin;
+package pro.taskana.simplehistory;
 
 import javax.annotation.PostConstruct;
-import org.springframework.beans.factory.annotation.Autowired;
+
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+import pro.taskana.configuration.TaskanaEngineConfiguration;
+import pro.taskana.history.api.TaskanaHistory;
+import pro.taskana.simplehistory.impl.HistoryEventImpl;
+
 @Component
 @Transactional
 public class ExampleBootstrap {
 
-    @Autowired
-    private HistoryService historyService;
+    private TaskanaHistory historyService;
 
+    ExampleBootstrap(TaskanaEngineConfiguration taskanaEngineConfiguration, TaskanaHistory historyService) {
+        this.historyService = historyService;
+        this.historyService.initialize(taskanaEngineConfiguration);
+    }
 
     @PostConstruct
-    public void test() {
-        HistoryEvent newEvent = historyService.newHistoryEvent("", "", "");
-        historyService.createHistoryEvent(newEvent);
-        HistoryEvent insertedEvent = historyService.getHistoryEvent(newEvent.getId());
-        System.out.println("---------------------------> event inserted: " + insertedEvent.getId());
+    public void testBootstraping() {
```

Comment:
If this is a test, why don't we create an acceptance test instead? 

I understand, you've just refactored that, but wouldn't that be better?

Reply:
I just followed the same structure and naming as taskana-spring-example. This is not a test it is just a bootstrap example.  I will change the bootstraping example, I haven't done tests because right now the logic is covered by this bootstraping. I think that we can add tests after adding more functionality.